addons/kruise-rollout/metadata.yaml

```diff
@@ -1,5 +1,5 @@
 name: kruise-rollout
-version: 1.3.0
+version: 1.3.1
```

Comment:
Since this pull request represents a significant change, please update the version number to 1.4.0

Reply:
done